web_view_calendar_list/static/src/js/calendar_list_renderer.js

```diff
@@ -0,0 +1,154 @@
+odoo.define('web_view_calendar_list.CalendarListRenderer', function (require) {
+    "use strict";
+
+    var CalendarRenderer = require('web.CalendarRenderer');
+    var session = require('web.session');
+    var core = require('web.core');
+    var qweb = core.qweb;
+    var _t = core._t;
+
+    var scales = {
```

Comment:
@AdriaGForgeFlow The main reason for the function is this. We need to rewrite a constant item :cry: